scaladoc/src/dotty/tools/scaladoc/tasty/comments/markdown/SnippetFormattingExtension.scala

```diff
@@ -0,0 +1,40 @@
+package dotty.tools.scaladoc
+package tasty.comments.markdown
+
+import dotty.tools.scaladoc.snippets._
+
+import com.vladsch.flexmark.formatter._
+import com.vladsch.flexmark.parser._
+import com.vladsch.flexmark.ext.wikilink._
+import com.vladsch.flexmark.ext.wikilink.internal.WikiLinkLinkRefProcessor
+import com.vladsch.flexmark.util.ast._
+import com.vladsch.flexmark.util.options._
+import com.vladsch.flexmark.util.sequence.BasedSequence
+import com.vladsch.flexmark._
+
+object SnippetFormattingExtension extends Formatter.FormatterExtension:
```

Comment:
Do you mind adding a scaladoc comment explaining what this extension does? (I guess it annotates the code blocks with compiler/interpreter output, a bit like tut or mdoc?)

Reply:
Yes, it is providing rich html in place for markdown codeblocks. Will update the comment too